### PR TITLE
Docs pass

### PR DESCRIPTION
**Pull request by nhynes**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ jobs:
           command: cargo test --tests
       - run:
           name: Test messaging example
-          working_directory: examples/messaging
+          working_directory: examples/ez-message
```
Comment (willscott) [inline .circleci/config.yml]: 🤔

Comment (nhynes) [reply .circleci/config.yml]: aww. @armaniferrante approve this real quick?

File: memchain/src/lib.rs [hunk 2]
```diff
@@ -2,6 +2,7 @@
 #![feature(maybe_uninit)]
 
 mod block;
+#[cfg(features = "ffi")]
```
Comment (willscott) [inline memchain/src/lib.rs]: why?

Comment (nhynes) [reply memchain/src/lib.rs]: because I don't want to pollute statically linked libs with useless global symbols if they're not needed

**Pull request merged**